pkg/queue/lru_queue.go

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package queue
+
+import (
+	"container/list"
+	"sync"
+
+	"github.com/dragonflyoss/Dragonfly/pkg/errortypes"
+)
+
+type cQElementData struct {
```

Comment:
Yes, I am on @lowzj .
Maybe you could add more comments on the struct, so that we can make it more readable.

Reply:
done.